_data/errors.yml

```diff
@@ -118,4 +118,78 @@ oidc:
+      id: pkce-or-private-key
+      accordion-id: pkce_or_private_key
+      content: |
+        ##### Why it's happening
+        OpenID Connect authentications must use either a PKCE or a private_key_jwt authentication menthod. Both methods require specific values to be sent in `token` endpoint request.
+        ##### What to do:
+        If your integration is PKCE:
+        - Ensure you are sending the `code_verifier` attribute as part of your token request. The value of this attribute is the original value of the `code_challenge` parameter sent in your original authentication request.
+        If your integration is private_key_jwt:
+        - Ensure you are sending `client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer` as part of your token request.
+
+        More information about token request values can be found here: [https://developers.login.gov/oidc/token/#request-parameters](https://developers.login.gov/oidc/token/#request-parameters){:target="_blank"}.
+    - title: Code is invalid because doesn’t match any user. Please see our documentation at https://developers.login.gov/oidc/#token
+      id: invalid-code
+      accordion-id: invalid_code
+      content: |
+        ##### Why it's happening
+        The `code` value being passed in the token request does not match any users. This typically happens when the code value is being reused in multiple token requests, or because the `code` value is not being passed in the token request.
+        ##### What to do:
+        - Ensure you are passing a `code` parameter in your token request.
+        - Ensure there is no possibility of the `code` value in the token request being sent multiple times.
+
+        Reasons can include:
+        - Browser is forcing an unexpected refresh
+        - User refreshes browser
+    - title: code_verifier did not match code_challenge
+      id: invalid-code-verifier
+      accordion-id: invalid_code_verifier
+      content: |
+        ##### Why it's happening
+        If you are using the PKCE protocol, the `code_verifier` value being passed in the **token** request does not match the decoded `code_challenge` value passed in the **authentication** request.
+        ##### What to do:
+        - Ensure that the `code_challenge` parameter is a URL-safe Base64 encoding of the SHA256 digest of the `code_verifier` parameter.
+    - title: iat must be an integer or floating point Unix timestamp representing a time in the past
+      id: invalid-iat
+      accordion-id: invalid_iat
+      content: |
+        ##### Why it's happening
+        The `iat` value in the `client_assertion` JWT is the time at which the JWT was issued. It takes the format of an integer timestamp representing the number of seconds since the Unix Epoch. This error means that it is:
+        - Not an integer or floating point Unix timestamp, OR
+        - It is not an integer/timestamp that is in the past.
+        ##### What to do:
```

Comment:
This header is showing up smaller than the rest of the What to do headers.

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/393/commits/cf8850248cc9e78c13667ec2062a5d7e12dd8aa8